__tests__/plants/getPlantsOfUser.mjs

```diff
@@ -0,0 +1,50 @@
+import { describe, it, expect } from "@jest/globals";
+import * as client from "../../src/clients/clients.mjs";
+import { initializeDatabase } from "../../src/database/initializeDatabase.mjs";
+import { clearDatabase } from "../../src/database/clearDatabase.mjs";
+import {
+  objectStatusCodes,
+  objectMessages,
+  objectData,
+} from "../../src/lib/getPlants/getPlantsObjects.mjs";
+
+describe("Get plants of user", () => {
```

Comment:
Algemeen goede testen alleen je vergelijkt niets, ik zou de getAllPlant endpoint gebruiken om een vergelijken te maken. Dit bespreken we morgen even voor de rest ziet het er goed uit!